docs/source/development/design/core.md

```diff
@@ -144,12 +144,12 @@ values directly to avoid having to create NetCDF files with trivial variables.
 
 ```toml
 [core.data.precipitation]
-file = /path/to/netcdf.nc
-var = prec
+file = "/path/to/netcdf.nc"
+var = "prec"
```

Comment:
Is it not valid toml if you don't have the quotes or something?

Reply:
Yeah any string in `toml` is supposed to be in quotes, and `sphinx` won't format this block as `toml` for the docs when this rule is broken. Looking at it again though I do think that `file` should be a literal string (') rather than a basic string (")

Reply:
Sounds good. Just checking!